doc/api/code_integrity.md

```diff
@@ -0,0 +1,117 @@
+# Code Integrity
+
+<!-- type=misc -->
+
+> Stability: 1.1 - Active development
+
+Code integrity refers to the assurance that software code has not been
```

Comment:
I think it should be opted-in for - behind a flag - to users. This documentation should first describe what they should do in the terminal such as _starting nodejs with --code-integrity_ then describing nuances of the code integrity.